zscaler/plugin.spec.yaml

```diff
@@ -75,3 +75,26 @@ actions:
         description: Information about given URLs
         type: "[]url_categorization"
         required: true
+  blacklist_url:
+    title: Blacklist URL
+    description: Blacklist or unblacklist a URL
+    input:
+      urls:
+        title: URLs
+        description: A given set of one or more URLs to update in the blacklist
+        type: "[]string"
+        required: true
+        example: ["example.com", "rapid7.com"]
```

Comment:
These are domains, does this allow URLs? e.g. https://example.com

Reply:
In API they call it "url", but I test it and this endpoint accepts only domains. I will change description then for:
"A given set of one or more domain name to update in the blacklist"

Reply:
@r7-kszczepanskagorna Gotcha, sorry, I just thought of a better way that would allow us to keep the same language as their API.

We can allow users to pass a URL like this: https://www.google.com or https://example.com/file.txt and we strip the uri prefix and the `/.*` suffix and then the action can handle both domains and URLs.

https://www.google.com -> google.com
 https://example.com/file.txt -> example.com

Then we can change the language back to URL in the descriptions